src/shared/components/timer/timer.component.js

```diff
@@ -0,0 +1,123 @@
+import React, { PureComponent } from 'react';
+import moment from 'moment';
+import firebase from 'firebase';
+
+import { StartButton, SaveButton, Time, HistoryContainer } from './timer.styles';
+
+require('firebase/firestore');
+
+export class Timer extends PureComponent {
```

Comment:
Może function component + hooki? Co sądzisz?

Reply:
I może sam`timer` powinien być hookiem? Gdybyś używał `function components` to byłoby to całkiem miłe w użytkowaniu. ;)